pkg/reconciler/volumeclaim/pvchandler.go

```diff
@@ -62,10 +62,10 @@ func (c *defaultPVCHandler) CreatePersistentVolumeClaimsForWorkspaces(ctx contex
 		switch {
 		case apierrors.IsNotFound(err):
 			_, err := c.clientset.CoreV1().PersistentVolumeClaims(claim.Namespace).Create(ctx, claim, metav1.CreateOptions{})
-			if err != nil {
+			if err != nil && !apierrors.IsAlreadyExists(err) {
 				errs = append(errs, fmt.Errorf("failed to create PVC %s: %s", claim.Name, err))
 			}
-			if err == nil || !apierrors.IsAlreadyExists(err) {
+			if err == nil || apierrors.IsAlreadyExists(err) {
```

Comment:
Nit: the error log here is confusing after we change the condition here: it suggests that this PVC is newly created even though the error is "Already Exists"

Reply:
@cugykw - I agree with @XinruZhang that it's confusing, do you mind taking another look 🙏🏾

Reply:
@XinruZhang @jerop Thanks for your comments, I tried to adjust it. PTAL again.

Reply:
thanks @cugykw, it's much clearer to me!